src/platforms/zulip/triagebot.md

```diff
@@ -0,0 +1,257 @@
+
+It can also be edited via Zulip by [private-messaging triagebot](https://rust-lang.zulipchat.com/#narrow/pm-with/261224-triage-rust-lang-bot). Any Rust organization member can edit their notifications page, or pages of other Rust organization team members. To do so, the editor must have a `zulip-id` listed in their people/username.toml file in the [team repository](https://github.com/rust-lang/team/). The bot will tell you which ID to use when talking to it for the first time; please `r? @Mark-Simulacrum` on PRs adding Zulip IDs.
+
+The following commands are supported:
+
+ * `acknowledge <url>`
+ * `acknowledge <idx>`
+
+These both acknowledge (and remove) a notification from the list.
+
+ * `add <url> <description... (multiple words)>`
+
+This adds a new notification to the list.
+
+ * `move <from> <to>`
+
+This moves the notification at index `from` to the index `to`.
+
+ * `meta <idx> <metadata...>`
+
+This adds some text as a sub-bullet to the notification at `idx`. If the metadata is empty, the text is removed.
+
+ * `as <github username> <command...>`
+
+This executes any of the above commands as if you were the other GH user.
+
+## Ping a team
+
+The bot can be used to "ping" teams of people that do not have corresponding Github teams. This is useful because sometimes we want to keep groups of people that we can notify but we don't want to add all the members in those groups to the Github org, as that would imply that they are members of the Rust team (for example, Github would decorate their names with "member" and so forth). The compiler team uses this feature to reach the [notification groups](https://rustc-dev-guide.rust-lang.org/notification-groups/about.html).
+
+When a team is pinged, we will both post a message to the issue and add a label. The message will include a `cc` line that `@`-mentions all members of the team.
+
+
+### Teams that can be pinged
+
+To be pinged, teams have to be created in the [Rust team repository](https://github.com/rust-lang/team). Frequently those teams will be marked as `marker-team`, meaning that they do not appear on the website. The [LLVM team](https://github.com/rust-lang/team/blob/master/teams/icebreakers-llvm.toml#L2) is an example.
+
+### Configuration
+
+To enable the team (e.g. `TeamName`) to be pinged, you have to add section to the `triagebot.toml` file at the root of a repository, like so:
+
+```toml
+[ping.TeamName]
+message = """\
+Put your message here. It will be added as a Github comment,
+so it can include Markdown and other markup.
+"""
+label = "help wanted"
+```
+
+This configuration would post the given message and also add the label `help wanted` to the issue.
+
+You can also define aliases to add additional labels to refer to same target team. Aliases can be useful to add mnemonic labels or accomodate slight mispellings (such as "llvms" instead "llvm"), see the following example:
+
+```toml
+[ping.cleanup-crew]
+alias = ["llvm", "llvms"]
```

Comment:
llvm is probably not an alias for cleanup crew?

Reply:
Yeah this is a copy paste error, those are the aliases for the llvm notify group.

Reply:
what would some good aliases example be? I don't have completely clear how they work

Reply:
Basically, without aliases you'd need to write:
```text
@rustbot ping cleanup-crew
```

Aliases allow you to replace `cleanup-crew` with other strings, so this is equivalent to the above command:

```text
@rustbot ping cleanup
```

I think you can pick the aliases for cleanup in the rust repo. Those include aliases like cleanup, bisection, mcve, etc...

Reply:
ok, commit https://github.com/rust-lang/rust-forge/pull/394/commits/24d108a3e2a9bf355abbf2f5f554393d39ec8bea should help now clarify this